include/defs/raster.h

```diff
@@ -40,6 +40,7 @@ int Rast_has_band_reference(const char *, const char *);
 int Rast_read_band_reference(const char *, const char *, char **, char **);
 int Rast_write_band_reference(const char *, const char *, const char *);
 int Rast_remove_band_reference(const char *);
+int R_legal_band_name(const char *);
```

Comment:
Why not `Rast_legal_band_name()` ?

Reply:
Forget about this. This is a bad idea. I am now playing with moving band IDs to map metadata block (a.k.a. struct History) + making sensor JSON file optional. Such approach would make band IDs tightly integrated and easy to use.